File: backend/Models/CreateFloorWithBookingPlacesRequest.cs
using BookingApp.Enums;

namespace BookingApp.Models
{
    public class CreateFloorWithBookingPlacesRequest
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Svg { get; set; }
        public List<BookingPlaceViewModel> BookingPlaces { get; set; }
    }
}

File: frontend/src/app/api-generated/models/create-floor-with-booking-places-request.ts
/* tslint:disable */
/* eslint-disable */
import { BookingPlaceViewModel } from '../models/booking-place-view-model';
export interface CreateFloorWithBookingPlacesRequest {
  bookingPlaces?: Array<BookingPlaceViewModel> | null;
  description?: string | null;
  name?: string | null;
  svg?: string | null;
}
